resources/core/charts/service-catalog/charts/etcd/values.yaml

```diff
@@ -1,40 +1,7 @@
-# Replicas scales the cluster with the "helm upgrade" command
-# The recommended etcd cluster size is 3, 5 or 7,
-# which is decided by the fault tolerance requirement.
-# A 7-member cluster can provide enough fault tolerance in most cases.
-# While larger cluster provides better fault tolerance the write performance
-# reduces since data needs to be replicated to more machines.
-replicaCount: 3
```

Comment:
we need to revert removing this field because when I installed it locally then replica count was 1

Reply:
I think replicaCount is set to 1 here: https://github.com/kyma-project/kyma/blob/master/resources/core/values.yaml#L7